x-pack/plugins/upgrade_assistant/__jest__/client_integration/es_deprecations/reindex_deprecation_flyout.test.ts

```diff
@@ -18,7 +18,6 @@ const defaultReindexStatusMeta: ReindexStatusResponse['meta'] = {
   aliases: [],
 };
 
-// Note: The reindexing flyout UX is subject to change; more tests should be added here once functionality is built out
```

Comment:
There have been additional tests added since this comment was first written, so I think it makes sense to remove.